usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -1314,23 +1342,62 @@ CK_RV token_specific_set_attrs_for_new_object(STDLL_TokData_t *tokdata,
                 add_pkey_extractable = CK_TRUE;
             break;
         }
-
         if (add_pkey_extractable) {
```

Comment:
Could we move that if block to after the switch statement? It appears in pretty much all cases.
If it would be behind the switch, it would just look at add_pkey_extractable as set by the switch cases and do its work or not, dependent on add_pkey_extractable.